### ISSUE TITLE
Fixed - Audi A8, ongoing V2.0.2 issues

### ISSUE THREAD

**Issue by crostonblue** [cpebit/aawireless-issues#154]
Hi,

Reported this issue a few days ago and done as suggested.

I've also done a factory reset, deleted the App, removed connection from my MMI, then started again as if the AA unit was brand new - same result, still dropping out.

How do I go back to V2.0.1, as that version worked perfectly.

Device - AAWC4o7FYC5
Car - 2021 Audi A8
Mobile - Samsung Galaxy S20 Ultra.

**Comment by Snirpo** [comment]
Enabled. Please give feedback if it actually helped.

**Comment by crostonblue** [comment]
Hi,

Installed v2.0.1 this morning and no reboots or issues on a 1 hour, 45 mile
drive.

Everything working perfectly again.

Thank you.



Regs David Brandon

**Comment by crostonblue** [comment]
Re-installed v2.0.1 this morning and no reboots or issues on a 1 hour, 45 mile drive.

Thank you.

**Comment by Snirpo** [comment]
@crostonblue Are you ok with upgrading to 2.0.2 again for a sanity check? I'll move it back to 2.0.1 afterwards.

**Comment by crostonblue** [comment]
Hi,

For awareness I started getting drop outs and failure to connect with
v2.0.1 as well, I was going to give up but I've done a few things that
seemed to have solved the issue.

What I found was that when the AA unit update takes place the Audi MMI
system in my 2021 Audi A8 sees the AA device as a new source and adds it as
a new connection - I had 4 AA connections in my MMI which may have been
affecting the connection.

I deleted all connections from the MMI, deleted the AA app from my android
phone and then did a factory reset of the AA unit before restarting as a
new device.

No drop outs or connection issues for the last few days.


Regs David Brandon
